supported-os-installer.sh

```diff
@@ -0,0 +1,524 @@
+#!/usr/bin/env bash
+
+# This script is mostly based on ->
+# https://github.com/Zimbra/zinstaller/blob/main/zinstaller script
+
+# Warnings :	This script is still a WORK IN PROGRESS. Not fully tested.
+#		Script to install Zimbra Classic - ONLY SINGLE NODE on a provided remote server
+############################################################################################
```

Comment:
should we update this now?

Reply:
Yes, Removed comments which are not required now!